NetTopologySuite/Operation/Union/UnionInteracting.cs

```diff
@@ -44,7 +44,7 @@ public IGeometry Union()
             // check for all interacting or none interacting!
             IGeometry int0 = ExtractElements(_g0, _interacts0, true);
             IGeometry int1 = ExtractElements(_g1, _interacts1, true);
-#if !PCL
+#if DEBUG

```

Comment:
Can `IsEmpty` throw, and/or can either `int0` or `int1` be `null`?  If the answer to either is true, then this is technically a breaking change; if so, does that matter?